docs/UserGuide.md

```diff
@@ -57,7 +57,7 @@ The following section will explain the steps taken to get **CLIrcuit Assistant**
 
 1. Ensure that you have Java 11 or above installed.
 1. Download the latest version of `Duke` from [the releases](https://github.com/AY2021S1-CS2113T-W13-3/tp/releases).
-1. Double click `jar` file to start the application. 
+1. Double click the `jar` file to start the application. 
```

Comment:
Everything seems fine and LGTM. However could you try putting this in the <style> </style> block at the top and see if it works? (unless of course if you have tried it out already) 
```css
@media print {
    div, image, svg {
        break-inside: avoid;
    }
```